rero_mef/agents/mef/api.py

```diff
@@ -217,12 +217,29 @@ def get_updated(cls, data):
         :param pid: pid to use..
         :returns: latest record.
         """
-        search = AgentMefSearch()
+        search = AgentMefSearch()\
+            .params(preserve_order=True) \
+            .sort({'pid': {'order': 'asc'}})
+        deleted = []
         if from_date := data.get('from_date'):
             search = search.filter('range', _updated={'gte': from_date})
         if pids := data.get('pids'):
+            # find deleted pids
+            all_pids = [hit.pid for hit in search.source('pid').scan()]
+            for missing_pid in sorted(list(set(pids) - set(all_pids))):
+                data = {'pid': missing_pid}
+                mef = AgentMefRecord.get_record_by_pid(
+                    missing_pid,
+                    with_deleted=True
+                )
+                if mef == {}:
```

Comment:
`if not mef`? Or better `if mef.is_deleted` -> https://github.com/inveniosoftware/invenio-records/blob/ce7989d84e418c9aed7d4e0bfcaffba8a10f0253/invenio_records/api.py#L105

Reply:
Does not give the same result!